src/basilisp/contrib/pytest/testrunner.py

```diff
@@ -29,7 +29,7 @@ def pytest_configure(config):
 def pytest_collect_file(file_path: Path, path, parent):
     """Primary PyTest hook to identify Basilisp test files."""
     if file_path.suffix == ".lpy":
-        if file_path.name.startswith("test_") or file_path.name.endswith("_test"):
+        if file_path.name.startswith("test_") or file_path.name.endswith("_test.lpy"):
```

Comment:
I think we can just check `file_path.stem.endswith("_test")`

Reply:
I've tried this but unfortunately file_path appears to be a `string` rather than `Path`, and as such abandoned that idea, given also that the there is an `if` statement testing for the `.lpy` extension ...

Reply:
let me try again ...

Reply:
done.